src/components/Gesture.svelte

```diff
@@ -235,14 +274,32 @@
             <i class="w-full h-full m-0 mt-4 p-2 fas fa-check fa-2x text-secondary" />
           </div>
           <StandardButton
-            onClick={recordClicked}
+            onClick={() => {
+              showCountdown = true;
+              countdownStart()}}
             small
             shadows={false}
             outlined
             fillOnHover>{$t('content.data.record')}</StandardButton>
         </div>
       {/if}
     </GestureTilePart>
+
+    {#if showCountdown === true}
```

Comment:
This `if` can be removed, as you have the `isOpen` property that does the same thing.